.dockerignore

```diff
@@ -5,6 +5,6 @@ test/
 .editorconfig
 .travis.yml
 appveyor.yml
-node_modules\
-dist\
-node_modules
\ No newline at end of file
+node_modules/
+dist/
+node_modules
```

Comment:
Do we need this with and without the slash?

Reply:
I don't think so. But I was just looking to correct the error I was running into and didn't know if there was some history behind having them in there both ways.

Reply:
Good point, I didn't see this was there already